components/submissions/SubmissionPrescriptionBox.tsx

```diff
@@ -0,0 +1,83 @@
+import { PDFDownloadLink } from "@react-pdf/renderer";
+import { FC } from "react";
+import { Status, Submission } from "../../interfaces";
+import { PDFDocSubmission } from "../../src/pdf/PDFDocSubmission";
+import {
+  AttachmentIcon,
+  DownloadIcon,
+  LoadingIcon,
+  NoPrescriptionIcon,
+  UploadIcon,
+} from "../icons";
+
+interface SubmissionPrescriptionBoxProps {
+  type: "upload" | "download";
+  submission: Submission;
+  prescription: string;
+  uploading: boolean;
+  uploadPrescription: (file: File) => void;
+}
+
+export const SubmissionPrescriptionBox: FC<SubmissionPrescriptionBoxProps> = ({
+  type,
+  submission,
+  prescription,
+  uploading,
+  uploadPrescription,
+}) => {
+  const onChange = (e: any) => {
```

Comment:
ok so, I don't recommend using any ... anywhere. To take advantage of TS type inference you can simply define the onChange handler inline and not specify any types yourself.

In this case it would be:

```
onChange={(e) => {
    const file = e.target.files[0];
    uploadPrescription(file);
}}
```